lib/settings.dart

```diff
@@ -66,12 +74,17 @@ class _SettingsState extends State<Settings> {
     return TableRow(
```

Comment:
❓ I know its not your changes, but why are we using table row here instead of our tto_table?